### PR TITLE
[ADD]Airbrake-js installtion instruction in svelte framework

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

File: packages/browser/examples/svelte/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Usage with Svelte
+
+Integration with Svelte is as simple as adding hooks (Server and Client) of handleError:
+
+- For handle error of server add handleError of HandleServerError type
+- For handle error of client add handleError of HandleClientError type
```
Comment (kyrylo) [inline packages/browser/examples/svelte/README.md:6]:
```suggestion
- For client error handling, add `handleError` of `HandleClientError` type
```

File: packages/browser/examples/svelte/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+    }
+  }
+}
+
+export {};
+```
+
+## Server Hook
+
+To create an error handler for server with a `Notifier` initialized with your `projectId` and `projectKey`. In this example the handler will be in a file called `src/hooks.server.js`.
```
Comment (kyrylo) [inline packages/browser/examples/svelte/README.md:32]: The first sentence sounds incomplete.

File: packages/browser/examples/svelte/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  return {
+    message: error,
+    errorId,
+  };
+}
+```
+
+## Client Hook
+
+To create an error handler for client with a `Notifier` initialized with your `projectId` and `projectKey`. In this example the handler will be in a file called `src/hooks.server.js`.
```
Comment (kyrylo) [inline packages/browser/examples/svelte/README.md:67]: The first sentence sounds incomplete.

File: packages/browser/examples/svelte/README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  return {
+    message: error,
+    errorId,
+  };
+}
+```
+
+## Test
+
+To test that server hook has been installed correctly in your Svelte project, just open up the JavaScript console in your internet browser and paste in:
```
Comment (kyrylo) [inline packages/browser/examples/svelte/README.md:102]: Oh, you could use imports inside dev tools? I had no idea 😅

Comment (smurf-U) [reply packages/browser/examples/svelte/README.md:102]: :stuck_out_tongue:

**Comment by smurf-U** [comment]
PTAL

**Review by kyrylo — APPROVED** [review]

**Pull request merged**